### PR TITLE
fix(vjsstandard): update to 8.0.2 and fixup linting

### PR DESCRIPTION
**Pull request by gkatsev**
## Description
Please describe the change as necessary.
If it's a feature or enhancement please be as detailed as possible.
If it's a bug fix, please link the issue that it fixes or describe the bug in as much detail.

## Specific Changes proposed
Please list the specific changes involved in this pull request.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by gkatsev** [comment]
At least locally, the following line failes with `no-undef` but when trying a similar construct on the [eslint demo page](https://eslint.org/demo/#eyJ0ZXh0IjoiZnVuY3Rpb24gZm9vKGZvbywgYmFyLCBiYXope1xuICAgIGNvbnNvbGUubG9nKGZvbywgYmFyLCBiYXopO1xufVxuXG5mb28oJ2ZvbycsXG4gICAgNSk7Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo1LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7ImNvbnN0cnVjdG9yLXN1cGVyIjoyLCJmb3ItZGlyZWN0aW9uIjoyLCJnZXR0ZXItcmV0dXJuIjoyLCJuby1jYXNlLWRlY2xhcmF0aW9ucyI6Miwibm8tY2xhc3MtYXNzaWduIjoyLCJuby1jb21wYXJlLW5lZy16ZXJvIjoyLCJuby1jb25kLWFzc2lnbiI6Miwibm8tY29uc3QtYXNzaWduIjoyLCJuby1jb25zdGFudC1jb25kaXRpb24iOjIsIm5vLWNvbnRyb2wtcmVnZXgiOjIsIm5vLWRlYnVnZ2VyIjoyLCJuby1kZWxldGUtdmFyIjoyLCJuby1kdXBlLWFyZ3MiOjIsIm5vLWR1cGUtY2xhc3MtbWVtYmVycyI6Miwibm8tZHVwZS1rZXlzIjoyLCJuby1kdXBsaWNhdGUtY2FzZSI6Miwibm8tZW1wdHktY2hhcmFjdGVyLWNsYXNzIjoyLCJuby1lbXB0eS1wYXR0ZXJuIjoyLCJuby1lbXB0eSI6Miwibm8tZXgtYXNzaWduIjoyLCJuby1leHRyYS1ib29sZWFuLWNhc3QiOjIsIm5vLWV4dHJhLXNlbWkiOjIsIm5vLWZhbGx0aHJvdWdoIjoyLCJuby1mdW5jLWFzc2lnbiI6Miwibm8tZ2xvYmFsLWFzc2lnbiI6Miwibm8taW5uZXItZGVjbGFyYXRpb25zIjoyLCJuby1pbnZhbGlkLXJlZ2V4cCI6Miwibm8taXJyZWd1bGFyLXdoaXRlc3BhY2UiOjIsIm5vLW1peGVkLXNwYWNlcy1hbmQtdGFicyI6Miwibm8tbmV3LXN5bWJvbCI6Miwibm8tb2JqLWNhbGxzIjoyLCJuby1vY3RhbCI6Miwibm8tcmVkZWNsYXJlIjoyLCJuby1yZWdleC1zcGFjZXMiOjIsIm5vLXNlbGYtYXNzaWduIjoyLCJuby1zcGFyc2UtYXJyYXlzIjoyLCJuby10aGlzLWJlZm9yZS1zdXBlciI6Miwibm8tdW5kZWYiOjIsIm5vLXVuZXhwZWN0ZWQtbXVsdGlsaW5lIjoyLCJuby11bnJlYWNoYWJsZSI6Miwibm8tdW5zYWZlLWZpbmFsbHkiOjIsIm5vLXVuc2FmZS1uZWdhdGlvbiI6Miwibm8tdW51c2VkLWxhYmVscyI6Miwibm8tdW51c2VkLXZhcnMiOjIsIm5vLXVzZWxlc3MtZXNjYXBlIjoyLCJyZXF1aXJlLXlpZWxkIjoyLCJ1c2UtaXNuYW4iOjIsInZhbGlkLXR5cGVvZiI6MiwiaW5kZW50IjoyLCJmdW5jdGlvbi1wYXJlbi1uZXdsaW5lIjoyfSwiZW52Ijp7ImJyb3dzZXIiOnRydWUsIm5vZGUiOnRydWUsImNvbW1vbmpzIjp0cnVlLCJzaGFyZWQtbm9kZS1icm93c2VyIjp0cnVlLCJ3b3JrZXIiOnRydWUsImFtZCI6dHJ1ZSwibW9jaGEiOnRydWUsImphc21pbmUiOnRydWUsImplc3QiOnRydWUsInBoYW50b21qcyI6dHJ1ZSwianF1ZXJ5Ijp0cnVlLCJxdW5pdCI6dHJ1ZSwicHJvdG90eXBlanMiOnRydWUsInNoZWxsanMiOnRydWUsIm1ldGVvciI6dHJ1ZSwibW9uZ28iOnRydWUsInByb3RyYWN0b3IiOnRydWUsImFwcGxlc2NyaXB0Ijp0cnVlLCJuYXNob3JuIjp0cnVlLCJzZXJ2aWNld29ya2VyIjp0cnVlLCJhdG9tdGVzdCI6dHJ1ZSwiZW1iZXJ0ZXN0Ijp0cnVlLCJ3ZWJleHRlbnNpb25zIjp0cnVlLCJlczYiOnRydWUsImdyZWFzZW1vbmtleSI6dHJ1ZX19fQ==) it succeeds:
```js
  removeRemoteTextTrack({track = arguments[0]} = {}) {
```

**Comment by gkatsev** [comment]
Depends on this PR https://github.com/videojs/eslint-config-videojs/pull/33

File: src/js/close-button.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Component from './component';
  */
 class CloseButton extends Button {
 
- /**
+  /**
```
Comment (misteroneill) [inline src/js/close-button.js]: I know this is WIP, but just thought I'd point out that this change has come up a few times. Not entirely sure what causes it.

Comment (gkatsev) [reply src/js/close-button.js]: I saw that but seems like this fixes it? The `/` is one space back the function name. So, technically, all the `*` lines should be one space forward.

Comment (gkatsev) [reply src/js/close-button.js]: yeah, Ideally everything is
```js
/**
 *
 *
 */
function foo() {}
```

But I don't really feel like fixing it up manually. We can just fix it forward whenever we see something misaligned.

**Pull request merged**